Packs/CheckPhish/Integrations/CheckPhish/CheckPhish_test.py

```diff
@@ -45,4 +32,26 @@ def test_reliability_in_get_result_checkphish(requests_mock, mocker, bad_disp, e
 
     assert demisto.results.call_args_list[0][0][0]['Contents'] == RESULTS
     assert demisto.results.call_args_list[0][0][0]['EntryContext']['DBotScore']['Reliability'] == DBotScoreReliability.B
-    assert demisto.results.call_args_list[0][0][0]['EntryContext']['DBotScore']['Score'] == expected_score
+    assert demisto.results.call_args_list[0][0][0]['EntryContext']['DBotScore']['Score'] == 3
+
+
+def test_bad_disp_param(requests_mock, mocker):
+    """
+        Given:
+            - The user bad_disp param which does not exist in the default
+        When:
+            - Running get_result_checkphish
+        Then:
+            - Verify reliability as excepted
+            - Verify dbot score is expected according the bad disposition
+    """
+    mocked_results = RESULTS
+    mocked_results['disposition'] = 'bad'
```

Comment:
why a `bad` disposition is a dbotsscore 0?, you would have to explain in the docstring.
I would expect that a bad one would result in a dbotscore of 3.
something is not clear here.